### ISSUE TITLE
Rotation doesn't work out of the canvas

### ISSUE THREAD

**Issue by fsalmasri** [heartexlabs/label-studio#6836]
If the bounding box is right at the edge of the image (canvas) we can't rotate it because it can't go beyond the canvas. 

Is it possible to fix this ?

**Comment by heidi-humansignal** [comment]
Hello,

Currently, Label Studio restricts annotations to stay within the image canvas, which affects the ability to rotate bounding boxes at the edges. Our development team is aware of this limitation and is working on improving the annotation tools to allow more flexibility in such scenarios.
In the meantime, you might consider the following workaround:

- **Adjust the Bounding Box Temporarily**: Try resizing or moving the bounding box slightly away from the edge to perform the rotation. After rotating, you can adjust it back to align with the edge of the image as needed.
- **Zoom Out the Image**: If possible, zooming out may provide additional space around the image, allowing you to rotate the bounding box without exceeding the canvas boundaries.
We understand that these are not perfect solutions, and we apologize for any inconvenience this may cause. Your feedback is valuable, and we'll prioritize enhancing this functionality in future updates of Label Studio.


Thank you,
Abu

> Comment by Abubakar Saad
> [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/12645248729)

**Comment by fsalmasri** [comment]
These are not solutions but thank you for taking this into consideration.

**Comment by heidi-humansignal** [comment]
Yes, those are more of workarounds. I'll see if I can find the a solution for this. I'll create a ticket for it as well.

Thank you,
Abu

> Comment by Abubakar Saad
> [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/12703718092)
